site/ko/guide/tensor.md

```diff
@@ -0,0 +1,303 @@
+# 텐서플로 텐서
+
+이름에서 알 수 있듯이, 텐서플로는 텐서를 포함한 계산을 정의하고 실행하는 프레임워크입니다.
+*텐서*는 벡터와 행렬을 일반화한 것이고 고차원으로 확장 가능합니다.
+내부적으로 텐서플로는 텐서를 기본 변수형의 n-차원 배열로 나타냅니다.
+
+텐서플로 프로그램을 작성할 때, 조작하고 전달하는 중요 객체는 `tf.Tensor` 입니다.
+`tf.Tensor` 객체는 결과적으로는 값으로 변환될 수 있는 부분적으로 정의된 계산으로 표현됩니다.
+텐서플로 프로그램은 `tf.Tensor` 객체 그래프를 만드는 것으로 먼저 시작하고,
+각각의 텐서가 다른 텐서를 기반으로 어떤 식으로 계산될 수 있는지 구체화하고,
+그 다음 그래프를 실행해서 원하는 결과를 얻게 됩니다.
+
+`tf.Tensor`는 다음과 같은 속성을 가지고 있습니다:
+
+ * 변수형 (예를 들어, `float32` 또는 `int32`, `string`)
+ * 형태
```

Comment:
'shape' can be translated as '형태' or '크기'. There are pros and cons in both of them.
'형태' is used to translate 'form' and '크기' is used to translate 'size'.
Buit I like '크기' because we often say to others '이 배열의 크기 얼마야?' rather than '이 배열 형태가 뭐야?'. 
But if you use '형태', I suggest to add english term. '형태' --> '형태(shape)'

Reply:
In my opinion, the meaning seems too simple to translate to '크기' because '크기(size)' is usually represented as a single numeric value. So, '형태(shape)' is used in this translation.